README.md

```diff
@@ -1090,7 +1092,17 @@ There are two css classes (`event` and `message`) that should be applied to mess
 
 #### Miro
 
-Another option is using [Miro](https://miro.com/). Miro allows the creation of diagrams in a whiteboard that can be exported and imported as a backup file (RTB) by a board owner. Diagrams created in Miro can be used to generate images that should be committed along with whiteboard backup. The idea is that the whiteboard with the diagrams can be imported in Miro by any team and evolved along with the documentation. To generate diagram images from Miro board, surround the diagram with a frame, select the frame, select the "Export as image" option and export the image as PNG.
+Another option is [Miro](https://miro.com/). Miro allows the creation of diagrams on a whiteboard. A board can be saved as a board backup file (.rtb) and can be used to create images.
+
+To create an image:
+
+- Drag a rectangle around the relevant objects on the board
+- From the “three dots” (…) menu above the selection:
+  - Click “Create frame”
+  - Click “Export as image”
+- Click the appropriate size of JPG and click "Export"
```

Comment:
I could find a PNG option, and the [docs](https://help.miro.com/hc/en-us/articles/360017572754-How-to-export-your-board) say that PNG isn't possible.

Reply:
Grateful that you spotted it @adamralph !

1. For the standard case, I forgot to include **_a very important_** step which was to copy the image to pain and then save it as a `.png` . 
 
2. There is an alternative way in which Miro will allow to export to .png and that is if the diagram is a draw.io diagram (which we also have [here](https://miro.com/app/board/uXjVNknXdYo=/?moveToWidget=3458764581646656814&cot=14)). You'd have to double-click the diagram and on edit mode export to `.png` (see picture below)
<img width="455" alt="image" src="https://github.com/Particular/docs.particular.net/assets/10582064/122ed9be-e501-4b43-9b0b-4ecd69deb489">

As an additional piece of information, we should steer away from generating images with transparent backgrounds as when users have a dark theme enabled, which can be a bigger proportion for engineers, the lines in the diagram can become hard to see therefore generating a bad user experience whilst reading our documentation.

Should we add this alternative way of doing it and highlighting this note about the dark theme on the docs?